soda-core/src/soda_core/common/extensions.py

```diff
@@ -11,7 +9,6 @@ def find_class_method(cls, module_name: str, class_name: str, method_name: str)
             module = import_module(module_name)
             class_ = getattr(module, class_name)
             return getattr(class_, method_name)
-        except AttributeError as e:
-            raise ExtensionException(
-                message=f"Feature '{class_name}.{method_name}' requires the Soda Extensions to be installed."
-            )
+        except (AttributeError, ModuleNotFoundError) as e:
+            # Extension not installed
```

Comment:
when would this happen, why do we change it to silently continue? I see the usage below, which tries to load the extension with a hardcoded module/class/method names, but that will be replaced by registering the extension using entry points. I would therefore keep the exception raising here and temporarily catch the exception and silently continue in the ContractVerificationHandler

Reply:
Removed extentions.py file and inlined the class instantiation into the calling class.